onnx/checker.cc

```diff
@@ -814,6 +799,30 @@ void check_model(const ModelProto& model) {
   check_model(model, ctx);
 }
 
+
+std::set<std::string> experimental_ops = {"ATen",
+                                          "Affine",
+                                          "ConstantFill",
+                                          "Crop",
+                                          "DynamicSlice",
+                                          "GRUUnit",
+                                          "GivenTensorFill",
+                                          "ImageScaler",
+                                          "ParametricSoftplus",
+                                          "Scale",
+                                          "ScaledTanh"};
+
+bool checkIsExperimental(std::string node_op_type) {
+  if (experimental_ops.count(node_op_type)) {
+    std::cerr << "Warning: " << node_op_type << " was a removed "
```

Comment:
This method as the name suggests should only check whether the given op is experimental or not... .i.e return true if it is or false if not... the calling code should decide what to do with the result. I would say move the error out of this method

Also in case of onnx checker the warning message should be something like - checker does not support models with experimental ops